### PR TITLE
Fixed examples for zig 0.7

### PR DESCRIPTION
**Pull request by Guigui220D**
Used {s} when printing strings

### CODE REVIEW

File: build.zig [hunk 1]
```diff
@@ -8,9 +8,15 @@ const EXAMPLES = .{
...
     const mode = b.standardReleaseOptions();
+    const target = b.standardTargetOptions(.{});
 
     const tests = b.addTest("src/sqlite.zig");
     tests.setBuildMode(mode);
+    //Added by me
+    tests.setTarget(target);
+    tests.addIncludeDir("../sqlite-c/");
+    tests.addLibPath("../sqlite-c/");
+
```
Comment (leroycep) [inline build.zig:19]: Is this for using versions of sqlite other than the one installed on the system?

**Review by leroycep — COMMENTED** [review]
Hey, sorry for the long wait, I didn't notice this until now. Looks good for the most part, I just have a question about the stuff in `build.zig`.

File: build.zig (hunk 1 above)
Comment (Guigui220D) [reply build.zig:19]: Oh i'm sorry I thought this wasnt pushed

Comment (Guigui220D) [reply build.zig:19]: I dont know if you can edit and remove those lines from the site (i'm on my phone); I didnt mean to push those

Comment (leroycep) [reply build.zig:19]: I'll see if I can remove the second commit with `git reset --hard HEAD~1` and then `git push --force`

**Comment by leroycep** [comment]
Okay, I went ahead and merged the first commit, but I left the second one off, mostly because of `tests.addIncludeDir` and `tests.addLibPath`. I'd be happy to add the other code if you want to submit another pull request or if you want me to edit and merge the code myself.

**Pull request closed without merging**